dapp/src/components/buySell/SwapCurrencyPill.js

```diff
@@ -16,8 +16,8 @@ import {
 import { assetRootPath } from 'utils/image'
 import { _ } from 'fbt-runtime/lib/fbt'
 
-const CoinImage = ({ small, coin, isSemiTransparent = false }) => {
-  const className = `coin-image ${isSemiTransparent ? 'transparent' : ''}`
+const CoinImage = ({ small, coin }) => {
+  const className = `coin-image`
```

Comment:
We no longer need to use backtick here because this is not a template literal any more.

We can also make use of Eslint quotes rule in this project?